client/hooks/useMuncher.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import newGame from '../GameState/muncherState.js';
+
+export function useMuncher() {
+  // I need to hand a new array to react
+  // will get moved to a hook eventually?
+  const [game, setGame] = useState(newGame);
+  const [targetSound, setTargetSound] = useState('ee');
+  const [gameover, setGameover] = useState(false);
+  const [matchesLeft, setMatchesLeft] = useState(4);
+  const [points, setPoints] = useState(0);
+  const [lives, setLives] = useState(['x', 'x', 'x']);
+  const [currentPosition, setCurrentPosition] = useState(0);
+
+  // get these targets from a fetch?
+  const targets = ['ee', 'ea', 'oo', 'ou'];
+
+  const checkMatches = () => {
+    // when resetting the phonics fetch a whole new game for those sounds
+    if (matchesLeft < 1) {
+      if (targetSound === 'ee') {
+        setTargetSound('ea');
+        setMatchesLeft(4);
+      }
+      else {
+        setTargetSound('ee');
+        setMatchesLeft(4);
+      }
+    }
+  };
```

Comment:
I think I follow the purpose of this function and how it works for the 'ee' and 'ea' sounds, but I also see that in your targets array you have two other sounds, and I was wondering when those come into play?

Reply:
They didn't, it was some data that wasn't being used at all. Before the game starts I will have the player pick their difficulty and target phonic group. It will pick a random phonic sound, and everytime I refetch some words it will use a different phonic sound from that group.